bindings/cpp/logger.cpp

```diff
@@ -13,11 +13,17 @@
 #include <blackhole/record.hpp>
 #include <blackhole/logger.hpp>
 
+#include "example/config.hpp"
 #include "elliptics/session.hpp"
 #include "library/elliptics.h"
 
 namespace ioremap { namespace elliptics {
 struct trace {
+	struct trace_stack_t {
```

Comment:
`std::stack` is an adapter and it uses underlying container, by default it is `std::deque`.